graylog2-web-interface/src/components/common/TypeAheadFieldInput.jsx

```diff
@@ -57,22 +65,23 @@ class TypeAheadFieldInput extends React.Component {
               },
             );
 
-            if (this.props.autoFocus) {
+            if (autoFocus) {
               fieldInput.focus();
               fieldInput.typeahead('close');
             }
           },
         );
 
+      // eslint-disable-next-line react/no-find-dom-node
       const fieldFormGroup = ReactDOM.findDOMNode(this.fieldInput);
```

Comment:
Isn't `this.fieldInput` the DOM Node already?

Reply:
I didn't investigate too deeply, was just clearing out lint errors and I didn't want to fully rebuild these components. Also, these changes aren't necessary for the CSS update, so I'm happy to revert them if we feel it makes more sense to just focus on the style updates in this PR.

Reply:
I'd leave it like this. I also ran into old components needing an update thus trigging a bunch of eslint errors. Suppressing is a good option in that case